src/app/core/components/upload-progress/upload-progress.component.html

```diff
@@ -13,7 +20,13 @@
     class="upload"
     *ngIf="status === UploadSessionStatus.InProgress && currentItem"
   >
-    <div class="current-file">{{ currentItem.file.name }}</div>
+    <div *ngIf="!isUploadingInNestedFolder" class="current-file">
+      {{ currentItem.file.name }}
+    </div>
+    <div *ngIf="isUploadingInNestedFolder" class="current-file">
+      Uploading {{ currentItem.file.name }} to
+      {{ currentItem.parentFolder.displayName }}
+    </div>
```

Comment:
@crisnicandrei I think this can be simplified. We can show the destination folder for all uploads, not just nested folders.